### DISCUSSION TITLE
Help with technical and hardware questions

### DISCUSSION THREAD

**Discussion by medapayne** [awawa-dev/HyperHDR#976] [General]
Good evening everyone

I allow myself to create this topic mainly for hardware and technical questions to avoid spending money as I have unfortunately been confronted with in the past.
I have tested a lot of Ambilight boxes like the Lytmy Pro / Fantasy3, Dreamscreen 4k and Govee Sync Box 2 and all have a problem of washed out colors in HDR despite the promises.

Technical questions:

- So doesn't HyperHdr have this problem of washed out or overly saturated colors in HDR?

- Is HyperHdr functional on the eArc, VRR and ALLM side? Depends on the hardware used?

Hardware questions:

- I mainly use HDR (not the 10 Samsung QLED TV), HDMI 2.1 and eArc, which Grabber and Splitter do you recommend to be able to use all this potential.

- I want to use multiple sources with HyperHdr like my Apple TV, my Switch and my PC which is currently connected to my television so could you recommend a 4K 120Hz eArc VRR and ALLM Splitter with 2 outputs and 3 or 4 inputs.

Sorry for all these novice questions and thank you in advance for your answers.

Ps: sorry for my English I use Google Translate

**Comment by awawa-dev** [comment]
Hi
For HDR10/Dolby Vision check these posts:
- latest modern grabber with experimental support for P010 in HyperHDR https://github.com/awawa-dev/HyperHDR/discussions/967#discussioncomment-11124448 These grabbers should support VRR but not HDCP
- classic cheap grabber MS2130: https://github.com/awawa-dev/HyperHDR/discussions/729 No VRR

I would check the offer of Ezcoo splitters/matrices because they are most reliable in the cheap segment on the market and most of the should supports even Dolby Vision (only Low Latency Dolby Vision) like EZ-SP12H2. They have also 4k120/VRR capable splitters but I didnt test them. I'm not sure about eArc & ALLM for these splitters...

**Reply by medapayne** [reply]
thank you very much for the answers
I am looking for a splitter with four inputs because I would like to use several sources what you propose only offers one. Would you have one to recommend to me?

and for this question please : HyperHdr have this problem of washed out or overly saturated colors in HDR?

**Reply by awawa-dev** [reply]
> and for this question please : HyperHdr have this problem of washed out or overly saturated colors in HDR?

You have a examples before and after with tone mapping in the links in the post above.

> Would you have one to recommend to me?

Unfortunately I haven't tested such a splitter/matrix so cannot recommend one. I can only recommend Ezcoo brand.

**Reply by medapayne** [reply]
thank you very much for the answers.
i will look at all this and come back if i need help

**Reply by medapayne** [reply]
Hello @awawa-dev 

I created a schema:

<img width="704" alt="Capture d’écran 2024-11-10 à 19 18 08" src="https://github.com/user-attachments/assets/c5bab1e2-34ee-4743-9b12-31f6ebf9890f">


can you tell me if it would work with two input sources for HDR (Apple TV) and 4K120Hz (gaming PC)

Here is the list of components I want to buy:

- grabber: https://fr.aliexpress.com/item/1005007598506671.html?spm=a2g0o.productlist.main.5.7294KchJKchJeY&algo_pvid=8623ef07-31e0-4f0f-918e-a29a172e90d1&algo_exp_id=8623ef07-31e0-4f0f-918e-a29a172e90d1-2&pdp_npi=4%40dis%21EUR%216.69%216.69%21%21%217.05%217.05%21%402101ef6817312597070382661ed974%2112000041449276114%21sea%21FR%21136508770%21X&curPageLogUid=TVIQAelckXzH&utparam-url=scene%3Asearch%7Cquery_from%3A
- Splitter: https://www.amazon.fr/Splitter-Sorties-HDCP2-3-R%C3%A9partiteur-Compatible/dp/B0C739QQ4Q/?_encoding=UTF8&pd_rd_w=mgyTh&content-id=amzn1.sym.cbda4761-bc4c-4d6a-919d-d7d29e1cd068%3Aamzn1.symc.9556efd4-e363-43c4-82ba-62279bb2c460&pf_rd_p=cbda4761-bc4c-4d6a-919d-d7d29e1cd068&pf_rd_r=TSCGTT9NXGZ2G72T7SDX&pd_rd_wg=vAsjD&pd_rd_r=8166b206-c213-48f3-ba10-ded0723549d5&ref_=pd_hp_d_atf_ci_mcx_mr_ca_hp_atf_d
- HDMI Switch: https://www.amazon.fr/UGREEN-Splitter-Commutateur-Compatible-Projecteur/dp/B0CPY5Y7Y3?crid=2EVTKEBX1NH0V&dib=eyJ2IjoiMSJ9.a2vBH75tTK0KqcAc_KBhloN23n8zCVN7fvyzDipBWObLURO2rYx8oL2f_0DpxPVq2a0rNA4tdPdaWMcrXcDCSHBaQ4GenqjuSxoM-T7NEsJG_UbjcsyMF8-iJ-0ggkeBHWm4uKhOJ8nXLCjrTKfCayny8RWPHT4rRh4qm670J4HgzkK050F70hye5rdjwUL9Df3IdlzA9czwzXToFZqxQNZPlH9A2MZtIgDEF_nlNKT-FJhj-OKUYiELokJVhNcciHVNwm2OFoFjaB1K9NX7Df12awoC3dEMSEqbgBH3CQI.a9wbmoePwNKH2yhE5wHCoN45e8hRUWZ87vTw5U5Rw2k&dib_tag=se&keywords=switch+hdmi+4k+120hz+hdr&nsdOptOutParam=true&qid=1731259906&sprefix=switch+hdmi+4k+hdr%2Caps%2C126&sr=8-8

or

https://www.amazon.fr/UGREEN-Commutateur-Entr%C3%A9es-Compatible-Projecteur/dp/B0BRSR8GNQ?crid=2EVTKEBX1NH0V&dib=eyJ2IjoiMSJ9.a2vBH75tTK0KqcAc_KBhloN23n8zCVN7fvyzDipBWObLURO2rYx8oL2f_0DpxPVq2a0rNA4tdPdaWMcrXcDCSHBaQ4GenqjuSxoM-T7NEsJG_UbjcsyMF8-iJ-0ggkeBHWm4uKhOJ8nXLCjrTKfCayny8RWPHT4rRh4qm670J4HgzkK050F70hye5rdjwUL9Df3IdlzA9czwzXToFZqxQNZPlH9A2MZtIgDEF_nlNKT-FJhj-OKUYiELokJVhNcciHVNwm2OFoFjaB1K9NX7Df12awoC3dEMSEqbgBH3CQI.a9wbmoePwNKH2yhE5wHCoN45e8hRUWZ87vTw5U5Rw2k&dib_tag=se&keywords=switch+hdmi+4k+120hz+hdr&nsdOptOutParam=true&qid=1731259906&sprefix=switch+hdmi+4k+hdr%2Caps%2C126&sr=8-5#customerReviews

Thanks in advance for your help

**Reply by awawa-dev** [reply]
I don't use such an extensive setup so I don't know. BTW you can host HyperHDR on PC and if you use it for gaming in Windows you don't even need RPi and HDMI switch because accelerated software grabber works very well and fast (DirectX11 grabber won't grab HDCP applications like Netflix on Windows though). In such case a lot of hardware would be unnecessary and only that poor AppleTV would be left which requires Ezcoo and the grabber.

**Reply by medapayne** [reply]
You are the best
I really like the idea of ​​installing HyperHDR on Windows, so I have several questions for Hardware and Software.
How to connect the LED strip?
What hardware is needed for the full installation?
I haven't found a tutorial for the full installation on windows, do you have a link to suggest to me please.
So I would do an installation via the Raspberry Pi for the Apple TV,
it may be stupid but I am really a novice: can I use the same LED strip as the one on Windows? or would I have to hang two on the back of my television?

I am really sorry for all these questions but I am really a novice so thank you in advance @awawa-dev

**Reply by awawa-dev** [reply]
HyperHDR runs on Windows as well and you can connect to PC a LED driver like Adafruit Scorpio to the USB port. Same as USB grabber. Or it can forward capture video stream to RPi: https://github.com/awawa-dev/HyperHDR/wiki/Quick-start#synchronization-between-different-hyperhdr-devices

**Reply by medapayne** [reply]
Wait but this is awesome!!!!

If I understand correctly on one side on my Windows PC I can install HyperHDR without anything else hardware level and on the other side I install HyperHDr on my PI with grabber and Splitter and led strip and HyperHDR of Windows will be able to send the information to the other HyperHdr of my PI so that the leds synchronize.... is that right @awawa-dev ?

**Reply by awawa-dev** [reply]
yes

**Reply by medapayne** [reply]
> yes

Thx @awawa-dev thank's alot for your answers
